docs/docs-beta/docs/guides/data-modeling/metadata.md

```diff
@@ -143,13 +156,13 @@ Dagster can automatically attach code references to your assets during local dev
 
 ### Attaching custom code references for local development
 
-You can manually add the `dagster/code_references` metadata to your asset definitions to attach custom code references. This is useful when your asset's source code is not primarily in Python (e.g. when parsing a YAML config or integrating with another language).
+You can manually add the `dagster/code_references` metadata to your asset definitions to attach custom code references. This is useful when your asset's source code isn't primarily in Python (e.g. when parsing a YAML config or integrating with another language).
```

Comment:
A few thoughts here:

- Should this be reframed to `Attaching non-Python code references`? I looked at this and wasn't sure what `custom` meant until I finished reading the intro paragraph
- Let's switch the sentences in this paragraph around so the use case comes before the solution. For example, something like:

   > If an asset's source code isn't primarily in Python - such as parsing a YAML config or integrating with another language - you can still attach code references. Manually attaching the `dagster/code_references` metadata to your asset definitions will allow you to link to the asset's source code.